functests/5_latency_testing/latency_testing.go

```diff
@@ -28,26 +30,27 @@ const (
 	//invalid values error messages
 	unexpectedError = "Unexpected error"
 	//incorrect values error messages
-	incorrectMsgPart1              = "the environment variable "
-	incorrectMsgPart2              = " has incorrect value"
-	maxInt                         = "2147483647"
-	mustBePositiveInt              = ".*it must be a positive integer with maximum value of " + maxInt
-	mustBeNonNegativeInt           = ".*it must be a non-negative integer with maximum value of " + maxInt
-	incorrectCpuNumber             = incorrectMsgPart1 + latencyTestCpus + incorrectMsgPart2 + mustBePositiveInt
-	nonpositiveCpuNumber           = incorrectMsgPart1 + latencyTestCpus + " has a nonpositive value" + mustBePositiveInt
-	incorrectDelay                 = incorrectMsgPart1 + latencyTestDelay + incorrectMsgPart2 + mustBeNonNegativeInt
-	negativeDelay                  = incorrectMsgPart1 + latencyTestDelay + " has a negative value" + mustBeNonNegativeInt
-	incorrectMaxLatency            = incorrectMsgPart1 + maximumLatency + incorrectMsgPart2 + mustBeNonNegativeInt
-	negativeMaxLatency             = incorrectMsgPart1 + maximumLatency + " has a negative value" + mustBeNonNegativeInt
-	incorrectOslatMaxLatency       = incorrectMsgPart1 + "\"" + oslatMaxLatency + "\"" + incorrectMsgPart2 + mustBeNonNegativeInt
-	negativeOslatMaxLatency        = incorrectMsgPart1 + "\"" + oslatMaxLatency + "\" has a negative value" + mustBeNonNegativeInt
-	incorrectCyclictestMaxLatency  = incorrectMsgPart1 + "\"" + cyclictestMaxLatency + "\"" + incorrectMsgPart2 + mustBeNonNegativeInt
-	negativeCyclictestMaxLatency   = incorrectMsgPart1 + "\"" + cyclictestMaxLatency + "\" has a negative value" + mustBeNonNegativeInt
-	incorrectHwlatdetectMaxLatency = incorrectMsgPart1 + "\"" + hwlatdetecMaxLatency + "\"" + incorrectMsgPart2 + mustBeNonNegativeInt
-	negativeHwlatdetectMaxLatency  = incorrectMsgPart1 + "\"" + hwlatdetecMaxLatency + "\" has a negative value" + mustBeNonNegativeInt
-	incorrectTestRun               = incorrectMsgPart1 + latencyTestRun + incorrectMsgPart2
-	incorrectRuntime               = incorrectMsgPart1 + latencyTestRuntime + incorrectMsgPart2 + mustBePositiveInt
-	nonpositiveRuntime             = incorrectMsgPart1 + latencyTestRuntime + " has a nonpositive value" + mustBePositiveInt
+	incorrectMsgPart1                  = "the environment variable "
+	incorrectMsgPart2                  = " has incorrect value"
+	invalidNumber                      = " has an invalid number"
+	maxInt                             = "2147483647"
```

Comment:
Use ```math.MaxInt```
```
maxInt := strconv.Itoa(math.MaxInt)
```
It can't be a constant but I think it's better than this magic number
BTW I see different values when I print it:
https://play.golang.org/p/eh0d8zDF5b-

Reply:
should be math.MaxInt32

Reply:
I have discussed the matter of changing this constant to math.MaxInt32 with Artyom , but that will require that all the dependent constants be changed to variables , so we decided to leave that as it is for now.